rabbit_events_test.go

```diff
@@ -16,7 +16,7 @@ var _ = Describe("RabbitEvents", func() {
 		It("Should send messages after quiescence", func() {
 			messagesSent := make([]string, 0)
 			var l sync.Mutex
-			mockSender := func(m []string, ri *RabbitIni) {
+			mockSender := func(m []string, x string, ri RabbitConfig) {
```

Comment:
Its kinda unfortunate that BatchSender takes in a RabbitConfig (Because the batcher has no need for a rabbit config other than passing it to this function).. We should rework that before the next version, just noting it here.